.github/workflows/build.yml

```diff
@@ -19,31 +18,8 @@ jobs:
     - uses: actions/checkout@v3
     - if: inputs.submodule
       run: git submodule update --init --depth 1 -- ${{ inputs.submodule }}
-    
-    - name: Generate cache key
-      if: inputs.submodule
-      id: cache-key
-      run: |
-        cd ${{ inputs.submodule }}
-        echo "::set-output name=key::${{ inputs.submodule }}-$(git rev-parse --short HEAD)"
-      
-    - uses: actions/cache@v3
-      if: inputs.submodule
-      with:
-        path: ${{ inputs.submodule || '.' }}/output/images/miyoo-cfw-*.img
-        key: ${{ steps.cache-key.outputs.key }}
-      id: cache
-
-    - name: retrieve ccache
```

Comment:
We are using that cache, for ccache bulidroot feature, which reduce build time from 1,5h to 40 minutes

Reply:
Oh 🤦